addons/website_event/controllers/main.py

```diff
@@ -353,6 +354,10 @@ def _create_attendees_from_registration_post(self, event, registration_data):
     @http.route(['''/event/<model("event.event"):event>/registration/confirm'''], type='http', auth="public", methods=['POST'], website=True)
     def registration_confirm(self, event, **post):
         registrations = self._process_attendees_form(event, post)
+        event_ticket_ids = {registration['event_ticket_id'] for registration in registrations}
+        event_tickets = request.env['event.event.ticket'].browse(event_ticket_ids)
```

Comment:
Can you just do "event_tickets = registrations.event_ticket_id" ? Seems odd to use sets for this.